index.d.ts

```diff
@@ -516,7 +516,7 @@ declare namespace grapesjs {
 
   interface TraitView {
     noLabel?: boolean;
-    templateInput?: string | ((options: { trait: Trait }) => string);
+    templateInput?: string;
```

Comment:
We can't remove the support of `templateInput` as a function as it's necessary for the current customization mechanism.